src/overlays/actors/ovl_Boss_Dodongo/z_boss_dodongo.cpp

```diff
@@ -80,6 +80,10 @@ static Color_RGBA8 dustPrimColor_95 = {255, 255, 100, 255};
 
 static Color_RGBA8 dustEnvColor_95 = {255, 100, 0, 255};
 
+static u8 originalLavaFloorTex[8192] = {
+#include "assets/EUR_MQD/overlays/ovl_Boss_Dodongo/lava_floor_lava.rgba16.inc.c"
+};
+
```

Comment:
This will not compile if the user has not extracted the EUR_MQD ROM.

Reply:
yup good catch was not intended for commit